distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -53,6 +55,21 @@ MOVE;$OPENHAB_USERDATA/config/org/eclipse/smarthome/core/i18nprovider.config;$OP
 REPLACE;org.eclipse.smarthome.core.i18nprovider;org.eclipse.smarthome.i18n;$OPENHAB_USERDATA/config/org/eclipse/smarthome/i18n.config
 
 [3.0.0]
+REPLACE;org_eclipse_smarthome_model;org_openhab_core_model;$OPENHAB_USERDATA/org.ops4j.pax.logging.cfg
+REPLACE;org.eclipse.smarthome.model;org_openhab.core.model;$OPENHAB_USERDATA/org.ops4j.pax.logging.cfg
+REPLACE;org_eclipse_smarthome;org_openhab;$OPENHAB_USERDATA/org.ops4j.pax.logging.cfg
+REPLACE;org.eclipse.smarthome;org.openhab;$OPENHAB_USERDATA/org.ops4j.pax.logging.cfg
+REMOVE;logger.paperui;$OPENHAB_USERDATA/org.ops4j.pax.logging.cfg
```

Comment:
New commands need to be interpreted by: 

https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/bin/update

for Linux and macOS

and

https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/bin/update.ps1

for Windows.

Reply:
Ok I gave it a try. But I'm a total noob on PowerShell. Can you validate and eventually fix the PS command I inserted please ?